### PR TITLE
Initial commit of package dependency force-directed graph

### PR DESCRIPTION
**Pull request by BetsyMcPhail**
See http://issues.osehra.org/browse/VIVIAN-237

### CODE REVIEW

**Comment by BetsyMcPhail** [comment]
PTAL @josephsnyder. This page uses the same data as the Circular Layout.

File: Visual/package_dep_graph.php [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+  links.forEach(function(d) {
+    linkedByIndex[d.source.index + "," + d.target.index] = 1;
+  });
+
+  function neighboring(a, b) {
+    return linkedByIndex[a.index + "," + b.index];
+  }
+
+  function toggleConnectedNodes() {
+    d = d3.select(this).node().__data__;
```
Comment (josephsnyder) [inline Visual/package_dep_graph.php]: Adding a     "if (d3.event.defaultPrevented) return; // ignore drag" will prevent a drag of the node from also selecting it.

File: Visual/package_dep_graph.php [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+    .text(function(d) { return d.name })
+    .style("stroke", "black")
+    .style("opacity", 0);
+
+  // Set-up tooltips
+  node.append("title")
+    .text(function(d) {
+      var text = "Name: " + d.name;
+      if (d.group) text += "\nGroup: " + d.group;
+      if (d.dependencies_count > 0) {
```
Comment (josephsnyder) [inline Visual/package_dep_graph.php]: Any chance of mirroring the tool tip of the first which breaks out the ones that are both?

https://github.com/OSEHRA-Sandbox/Product-Management/blob/master/Visual/vista_pkg_dep.php#L149

File: Visual/package_dep_graph.php (hunk 1 above)
Comment (BetsyMcPhail) [inline Visual/package_dep_graph.php]: Good suggestion!

File: Visual/package_dep_graph.php (hunk 2 above)
Comment (BetsyMcPhail) [inline Visual/package_dep_graph.php]: This is done in latest commit

**Pull request merged**